hidimstat/dcrt.py

```diff
@@ -0,0 +1,237 @@
+"""Implementation of distillation Conditional Randomization Test, by Liu et
+al. (2020) <https://arxiv.org/abs/2006.03980>. Currently only d0_CRT
+resampling-free is implemented.
+"""
+import numpy as np
+from joblib import Parallel, delayed
+from hidimstat.utils import _lambda_max, fdr_threshold
+from scipy import stats
+from sklearn.base import clone
+from sklearn.ensemble import (RandomForestRegressor,
+                              RandomForestClassifier)
+from sklearn.linear_model import (Lasso, LassoCV)
+from sklearn.preprocessing import StandardScaler
+
+
+def dcrt_zero(X, y, fdr=0.1, estimated_coef=None, Sigma_X=None, cv=5,
+              n_regus=20, max_iter=1000, use_cv=False, refit=False,
+              loss='least_square', screening=True, screening_threshold=1e-1,
+              scaled_statistics=False, statistic='residual', centered=True,
+              alpha=None, solver='liblinear', fdr_control='bhq', n_jobs=1,
+              verbose=False, joblib_verbose=0, ntree=100,
+              type_prob='regression', random_state=2022):
+
```

Comment:
Missing docstring

Reply:
Some parameters are unclear, e.g. `type_prob`, `n_regus`
Try to coerce to sklearn terminology whenever possible.